LICENSE.md

```diff
@@ -0,0 +1,13 @@
+All artwork in this repo is made available under the Linux Foundation trademark
```

Comment:
Can we instead say this: `All artwork in this repo that is provided by the Linux Foundation is made available under the Linux Foundation trademark`

IIUC, the logo in the repo has been _donated_ to CSI community per this email
https://groups.google.com/d/msg/container-storage-interface-community/3mu0y-JEslc/44LhYY1WBgAJ

Reply:
That seems reasonable until CSI is officially inducted in to CNCF.